platform/image_processor/src/processor/job/mod.rs

```diff
@@ -129,8 +129,9 @@ impl<'a, G: ImageProcessorGlobal> Job<'a, G> {
 			}
 		}
 		// job completion
+		tracing::info!("publishing job completion event to {}", self.job.task.callback_subject);
 		self.global
-			.nats()
+			.jetstream()
```

Comment:
This should be nats?

Reply:
I don't think so. Platform api uses jetstream to handle the callback. See [image_upload_callback.rs file](https://github.com/ScuffleTV/scuffle/blob/lennart/platform-profile-pictures/platform/api/src/image_upload_callback.rs#L24)